packages/koa-shopify-auth/src/auth/client/request-storage-access.ts

```diff
@@ -10,7 +10,7 @@ const requestStorageAccess = (shop: string, prefix = '') => {
           doesNotHaveStorageAccessUrl: "${prefix}/auth/enable_cookies?shop=${encodeURIComponent(
     shop,
   )}",
-          appTargetUrl: "/?shop=${encodeURIComponent(shop)}"
+          appTargetUrl: "${prefix || '/'}?shop=${encodeURIComponent(shop)}"
```

Comment:
I should have been more clear earlier, sorry. I meant in line 2 we can make the following change 

```diff
- const requestStorageAccess = (shop: string, prefix = '') => {
+ const requestStorageAccess = (shop: string, prefix = '/') => {
 ```

Reply:
gotcha, less conditions :) I made a change, please have a look